q2_alignment/plugin_setup.py

```diff
@@ -25,11 +25,11 @@
 plugin.methods.register_function(
     function=q2_alignment.mafft,
     inputs={'sequences': FeatureData[Sequence]},
-    parameters={'n_threads': Int % Range(1, None)},
+    parameters={'n_threads': Int % Range(0, None)},
     outputs=[('alignment', FeatureData[AlignedSequence])],
     input_descriptions={'sequences': 'The sequences to be aligned.'},
     parameter_descriptions={
-        'n_threads': 'The number of threads. (Use -1 to automatically use all '
+        'n_threads': 'The number of threads. (Use 0 to automatically use all '
```

Comment:
Did you check that this actually works with `mafft`? From the mafft help text:

```bash
--thread # :     Number of threads (if unsure, --thread -1)
```

According to my tests of `mafft`, this doesn't appear to work --- the only way to set to use all cores is to set -1. I believe we discussed a strategy yesterday to keep this parameter at 0, but to set it to -1 in the Action itself.

Reply:
Good catch. I thought I checked it but apparently, I missed it. Thanks for the heads up. ⚔️